distribution/test/itests/test-itests-alliance/src/test/java/org/codice/alliance/test/itests/AllianceAppsTest.java

```diff
@@ -1,99 +0,0 @@
-/**
```

Comment:
By adding the apps to the default list to start for the itests we already perform this test. No need to run it again.